### ISSUE TITLE
Can't open edit attack pop-up

### ISSUE THREAD

**Issue by tmarquesbr** [Zakarik/foundry-mm3#71]
I found this error when I was trying to edit the attacks. This error blocks me from open the pop-up no matter what. This happens with old and new actors. This is what my console returned (the Foundry version is V11):

```
 editAtk.mjs:28
 Uncaught (in promise) TypeError: undefined. Cannot read properties of undefined (reading 'localeCompare')
 [Detected 1 package: system:mutants-and-masterminds-3e(1.33.4)]
     at editAtk.mjs:28:36
     at Array.sort (<anonymous>)
     at get allStatus [as allStatus] (editAtk.mjs:28:13)
     at get html [as html] (editAtk.mjs:37:28)
     at editAtk.handleDialog (editAtk.mjs:332:115)
     at HTMLElement.<anonymous> (common.mjs:2883:10)
     at HTMLElement.dispatch (jquery.min.js:2:43184)
     at y.handle (jquery.min.js:2:41168)
```

**Comment by Zakarik** [comment]
Even by creating a new attack?

**Comment by tmarquesbr** [comment]
Unfortunately yes. Even if I create a new attack with an old actor or creating a new actor just to create a new attack triggers the same error.

**Comment by Zakarik** [comment]
I've sent updates, try to see if that solves anything.

If not, send me a character on which it doesn't work.

**Comment by tmarquesbr** [comment]
A good, a bad and an 'excellent' news.

Bad News: It didn't work, the error persists.

Good News: I created a dummy world. It's working, so the world itself has an error and it's on my side.

Excellent News: I ran "Find the Culprit" module at the world. At the first test, it solved, so it's a module conflict that the panel is not detecting. During 1.33 transition, the system did returned with several errors related to the actors' page. One of them was related to the system and one module, but I didn't capture the image, so I have to figure it out. This error, by the way, was fixed at 1.33.2
